account_wallet/views/wallet_views.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
nitpicking.

some part of this file are about various models. you could move the account.move.line view in the according file, (same for account.move)). 

So rename this file into ``account_wallet.xml``.